docs/managing-environments.mdx

```diff
@@ -0,0 +1,269 @@
+    When the agent succeeded and you're happy with the results:
+
+    ```bash
+    # Merge the environment into your current branch
+    cu merge fancy-mallard
+    
+    # Clean up (optional)
+    cu delete fancy-mallard
+    ```
+
+    The work becomes part of your Git history and you can continue from this new state.
+
+  </Tab>
+
+  <Tab title="🔄 Iterate & Refine">
+    When the agent is close but needs refinement:
+
+    ```bash
+    # Continue working in the same environment
+    # Prompt: "Work in the fancy-mallard environment and [refined instructions]"
+    ```
+
+    The agent will resume in the existing environment with all previous work intact. Perfect for:
+    - Adding missing features
+    - Fixing bugs the agent introduced
+    - Adjusting styling or behavior
+    - Building on partial progress
+
+  </Tab>
+
+  <Tab title="🗑️ Start Fresh">
+    When the agent went down the wrong path:
+
+    ```bash
+    # Discard the environment
+    cu delete fancy-mallard
+    
+    # Start over with a new prompt
+    # The agent will create a fresh environment from your current branch
+    ```
+
+    You're back to your last known good state (your current branch) and can try a different approach.
+
+  </Tab>
+</Tabs>
+
+## Resuming Work in Environments
+
+To continue work in an existing environment, simply mention it in your prompt:
+
+<CodeGroup>
+```text Good Resume Prompts
+"Work in the fancy-mallard environment and add user authentication"
+
+"Continue in environment fancy-mallard - the tests are failing, please fix them"
+
+"Using the fancy-mallard environment, add CSS styling to make it look modern"
+```
+
+```text What Happens
+The agent will:
+✅ Reuse the existing container state
+✅ Have access to all previous work
+✅ Continue from where it left off
+✅ Maintain the same dependencies and setup
+```
+</CodeGroup>
+
+## Practical Examples
+
+### Example 1: Happy Path Workflow
+
+```bash
+# 1. Agent creates environment and builds feature
+$ cu list
+ID            TITLE                    CREATED       UPDATED
+fancy-mallard Flask App with Login     2 mins ago    30 secs ago
+
+# 2. Quick check - looks good!
+$ cu diff fancy-mallard
++def login():
++    # Authentication logic
++    pass
+
+# 3. Accept the work
+$ cu merge fancy-mallard
+Updating abc123..def456
+Fast-forward
+ app.py | 15 +++++++++++++++
+ 1 file changed, 15 insertions(+)
+```
+
+### Example 2: Iteration Workflow
+
+```bash
+# 1. Check what agent built
+$ cu log fancy-mallard
+commit def456 (container-use/fancy-mallard)
+Author: Agent <agent@container-use.dev>
+Date:   Mon Jan 15 10:30:00 2024 -0800
+
+    Add basic login form
+    
+Notes (container-use):
+    $ flask run
+
+# 2. Needs improvement - continue in same environment
+# Prompt: "Work in fancy-mallard environment and add password validation"
+
+# 3. Check again after agent works
+$ cu diff fancy-mallard
+# Now shows both original work + new validation
+```
+
+### Example 3: Recovery Workflow
```

Comment:
i suspect "recovery" is the wrong word here, and we can maybe emphasize you don't actually _need_ to delete the env